hathor/manager.py

```diff
@@ -468,8 +476,10 @@ def _initialize_components(self) -> None:
                         assert tx.validate_full(skip_block_weight_verification=skip_block_weight_verification)
                         self.consensus_algorithm.update(tx)
                         self.tx_storage.indexes.update(tx)
-                        self.tx_storage.indexes.mempool_tips.update(tx)  # XXX: move to indexes.update
-                        self.step_validations([tx])
+                        if self.tx_storage.indexes.mempool_tips:
+                            self.tx_storage.indexes.mempool_tips.update(tx)  # XXX: move to indexes.update
+                        if self._enable_sync_v2:
+                            self.step_validations([tx])
```

Comment:
Should we rename it to `sync_v2_step_validations(...)`?

Reply:
I think that would make sense.

Reply:
Done.

Reply:
Thanks!